src/tests/TestHelper.ts

```diff
@@ -136,9 +137,9 @@ export module TestHelper {
         } else {
             let filename: string;
             if (inputFileOrScript.indexOf('import React') > -1) {
-                filename = 'file.tsx';
+                filename = Utils.absolutePath('file.tsx');
             } else {
-                filename = 'file.ts';
+                filename = Utils.absolutePath('file.ts');
```

Comment:
The tests previously just used 'file.ts(x)' as a mock filename. Since `reactNoDangerousHtmlRule` now compares `Utils.absolutePath(exception.file)` to `this.getSourceFile().fileName` (which also returns an absolute path when running tslint on an actual project) we need the tests to provide an absolute path for the filename, as well.

Unfortunately, this means I had to change a lot of test files, so this PR has a lot of line changes.